core/src/main/java/de/danoeh/antennapod/core/service/download/DownloadService.java

```diff
@@ -358,46 +359,42 @@ private void setupNotificationBuilders() {
     private Notification updateNotifications() {
         String contentTitle = getString(R.string.download_notification_title);
         int numDownloads = requester.getNumberOfDownloads();
-        String downloadsLeft;
-        if (numDownloads > 0) {
-            downloadsLeft = getResources()
-                    .getQuantityString(R.plurals.downloads_left, numDownloads, numDownloads);
-        } else {
-            downloadsLeft = getString(R.string.downloads_processing);
-        }
+        String downloadsLeft = (numDownloads > 0) ?
+                getResources().getQuantityString(R.plurals.downloads_left, numDownloads, numDownloads) :
+                getString(R.string.downloads_processing);
+
         if (notificationCompatBuilder != null) {
 
-            StringBuilder bigText = new StringBuilder("");
-            for (int i = 0; i < downloads.size(); i++) {
-                Downloader downloader = downloads.get(i);
-                final DownloadRequest request = downloader
-                        .getDownloadRequest();
-                if (request.getFeedfileType() == Feed.FEEDFILETYPE_FEED) {
-                    if (request.getTitle() != null) {
-                        if (i > 0) {
-                            bigText.append("\n");
+            List<String> lines = new ArrayList<>(downloads.size());
+            StringBuilder line = new StringBuilder("\u2022 ");
```

Comment:
It's also a good idea to test such changes on a device ;)
When I do changes on the UI, I usually to before&after screenshots. These can also be used to assert that no changes were made.